runway/hooks/staticsite/auth_at_edge/templates/sign_out/__init__.py

```diff
@@ -0,0 +1,72 @@
+
+# pylint: skip-file
+"""Sign the user out of Cognito and remove all Cookie Headers.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
+"""
+
+import logging
+from urllib.parse import urlencode
+
+from shared import extract_and_parse_cookies, get_config, get_cookie_headers  # noqa pylint: disable=import-error
+
+LOGGER = logging.getLogger(__name__)
+
+
+def handler(event, _context):
+    """Handle the signout event."""
+    request = event['Records'][0]['cf']['request']
+    config = get_config()
```

Comment:
Same as above, should be possible to instantiate get_config() outside the handler